src/components/Cards/MainProjectCard/index.tsx

```diff
@@ -1,10 +1,10 @@
 import * as S from "./styles";
 import { ViewIcons, PersonalIcons } from "../../../assets/icons";
-import { FeedList } from "../../../utils/api/Feed";
+import { project } from "../../../utils/api/Feed";
 import { FC } from "react";
 
 interface PropsType {
-  data: FeedList;
+  data: project;
```

Comment:
```suggestion
  data: Project;
```
타입이나 인터페이스는 대문자로 시작해주세요